mappings/net/minecraft/entity/passive/GroupingFishEntity.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS alz net/minecraft/entity/passive/GroupingFishEntity
```

Comment:
I think `school` would be more correct within the context of fish.

Reply:
Done.

Reply:
Maybe should apply to other contexts where `group` is used?